accesstoken/src/test/java/uk/gov/di/ipv/cri/common/api/handler/AccessTokenHandlerTest.java

```diff
@@ -20,7 +20,7 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import software.amazon.awssdk.http.HttpStatusCode;
 import uk.gov.di.ipv.cri.common.library.error.ErrorResponse;
-import uk.gov.di.ipv.cri.common.library.exception.AccessTokenValidationException;
+import uk.gov.di.ipv.cri.common.library.exception.*;
```

Comment:
you can remove the asterisk import and replace with individual import statements